tests/test_git/common.py

```diff
@@ -0,0 +1,60 @@
+import sublime
+import subprocess
+import os
+import shutil
+import tempfile
+from unittesting import DeferrableTestCase
+
+
+# copy from https://github.com/randy3k/ProjectManager/blob/f8667697a77a9e7020891e3227de25c2ebb990b3/pm.py#L72
+def subl(args=[]):
+    executable_path = sublime.executable_path()
+    if sublime.platform() == 'osx':
+        app_path = executable_path[:executable_path.rfind(".app/") + 5]
+        executable_path = app_path + "Contents/SharedSupport/bin/subl"
+    subprocess.Popen([executable_path] + list(args))
+    if sublime.platform() == "windows":
+        def fix_focus():
+            window = sublime.active_window()
+            view = window.active_view()
+            window.run_command('focus_neighboring_group')
+            window.focus_view(view)
+        sublime.set_timeout(fix_focus, 300)
```

Comment:
This feels like an implementation detail of `DeferrableTestCase`, can it be moved into a helper function in unittesting perhaps?